apps/web/modules/analysis/utils.tsx

```
@@ -26,3 +29,40 @@ export const renderHyperlinkedContent = (data: string): JSX.Element[] => {
+        surveyId: survey.id,
+        isEncrypted: survey.singleUse.isEncrypted,
+      });
+
+      if (singleUseIdResponse?.data) {
+        queryParams.push(`suId=${singleUseIdResponse.data}`);
+      } else {
+        const errorMessage = getFormattedErrorMessage(singleUseIdResponse);
+        throw new Error(errorMessage);
+      }
+    } catch (error) {
+      throw error;
```

Biome (1.9.4)

[error] The catch clause that only rethrows the original error is useless.

An unnecessary catch clause can be confusing.
Unsafe fix: Remove the try/catch clause.


(lint/complexity/noUselessCatch)